src/app.ts

```diff
@@ -66,9 +66,10 @@ app.use(
 app.use('/submission/configs', configRouter);
 app.use('/submission/program/:programId/registration', registrationRouter);
 app.use('/submission/program/:programId/clinical', submissionRouter);
-app.use('/submission/schema', schemaRouter);
```

Comment:
wont this be a breaking change for the apis used by gateway?  `list` `/template/*`

Reply:
I thought you wanted to deprecate the used ones first

Reply:
hmm, yea I was thinking of making the changes right after merging but I guess thats not happening yet. I'll put it back in for now